_posts/2019-02-24-The-Intelligent-Investor.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: post
+title:  "Book Review: The Intelligent Investor"
+date:   2019-02-24 22:40:05
+categories: jekyll update
+---
+
```

Comment:
I would start the article with a hook. What if I'm just reading your blog posts and don't know about `The Intelligent Investor` *a priori*.